fhd_core/gridding/visibility_grid.pro

```diff
@@ -94,6 +98,18 @@ psf_dim2=2*psf_dim
 psf_dim3=psf_dim*psf_dim
 bi_use_reduced=bi_use mod nbaselines
 
+if keyword_set(beam_per_baseline) then begin
+    uv_grid_phase_only=1
```

Comment:
As in `visibility_degrid`, please add a comment here explaining why this is always turned on.

Reply:
Done